openff/recharge/cli/reconstruct.py

```diff
@@ -48,7 +47,6 @@ def _retrieve_result_records(
         result.specification.keywords
         for result in client.query_records(
             record_id=record_ids,
-            limit=client.server_info["api_limits"]["get_records"],
```

Comment:
Do we need to do this query twice to get the keywords? Can we not just extract them later from the record?